patient_tracker/serializers.py

```diff
@@ -10,14 +23,21 @@ class Meta:
         model = Admission
 
         extra_kwargs = {'local_barcode': {'read_only': True}}
+        patient = PatientSerializer()
```

Comment:
I think new fields here should be marked as readonly (just like local_barcode), as it would break the admission creation

Reply:
Done!